src/o365/spo/commands/page/page-header-set.ts

```diff
@@ -447,10 +447,13 @@ class SpoPageHeaderSetCommand extends SpoCommand {
     functionality that isn't available on all tenants yet.
 
   Examples:
-  
+
     Reset the page header to default
       ${this.name} --webUrl https://contoso.sharepoint.com/sites/team-a --pageName home.aspx
```

Comment:
@appieschot , home.aspx can also be in quotes. Often the users have pages that have empty spaces in the name and if admin does not know and blindly follows the example, might be confused.